_api_app/app/Sites/Sections/Entries/Galleries/GalleryLinkRenderService.php

```diff
@@ -47,10 +49,9 @@ public function getViewData()
     public function getGalleryClassList()
     {
         $classes = parent::getGalleryClassList();
-        $galleryLinkAddress = !empty($this->entry['mediaCacheData']['@attributes']['link_address']) ? $this->entry['mediaCacheData']['@attributes']['link_address'] : '';
-        $galleryLinkTarget = !empty($this->entry['mediaCacheData']['@attributes']['linkTarget']) ? $this->entry['mediaCacheData']['@attributes']['linkTarget'] : '';
-        $classes[] = 'xGalleryLinkAddress-' . $galleryLinkAddress;
-        $classes[] = 'xGalleryLinkTarget-' . $galleryLinkTarget;
+        if (count($this->galleryItems) > 1) {
```

Comment:
couldn't you check this with `empty`? Boolean checks are usually better than counting elements.

Reply:
Nope, this checks if there is at least two items to able to add a hover effect.

Reply:
I see.